src/Parser/MarksmanshipHunter/Modules/Features/TimeFocusCapped.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
+
+import FocusTracker from '../FocusChart/FocusTracker';
+
+class TimeFocusCapped extends Module {
+  static dependencies = {
+    focusTracker: FocusTracker,
+  };
+
+  get getTotalWaste(){
+    let waste = this.focusTracker.secondsCapped * this.focusTracker.focusGen;
+    if (this.focusTracker.generatorCasts) {
+      waste += Object.values(this.focusTracker.activeFocusWasted).reduce((sum, waste) => sum + waste, 0);
+    }
+    return Math.round(waste);
+  }
+
+  statistic() {
+    const totalFocusWaste = this.getTotalWaste;
+    const percentCapped = formatPercentage(this.focusTracker.secondsCapped/(this.owner.fightDuration/1000));
+    return (
+      <StatisticBox
+        icon = {<Icon icon='ability_hunter_focusfire' alt = 'Focus Wasted' />}
+        label = 'Time Focus Capped'
+        tooltip= {`You wasted <b> ${totalFocusWaste}  </b> focus. <br /> 
```

Comment:
Nitpicking here, but I think indentation is key to making this readable. If the continued string is indented one more from the property, it stands out from the property list as being a continuation.